src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -117,18 +116,24 @@ const ResponseImpactDashboard: React.FC = () => {
   // set system wide data
   useEffect(() => {
     if (modelInputs.length === 0) return;
-    const localeDefaults = getLocaleDefaults(
-      localeDataSource,
-      modelInputs[0].stateCode,
-      modelInputs[0].countyName,
-    );
+
+    let incarceratedPopulation;
+    if (facilities.data[0].systemType === "State Prison") {
+      incarceratedPopulation = getLocaleDefaults(
+        localeDataSource,
+        modelInputs[0].stateCode,
+      ).totalPrisonPopulation;
+    } else {
+      incarceratedPopulation = getLocaleDefaults(
+        localeDataSource,
+        modelInputs[0].stateCode,
+        modelInputs[0].countyName,
+      ).totalJailPopulation;
+    }
```

Comment:
Ah, is this something I should add to my PR too? If we `getLocaleDefaults` and pass it a countyName for a "State Prison" system type will the `totalPrisonPopulation` be incorrect?

Reply:
Yes, if there is a `countyName` present but systemType is 'State Prison' then the population coming from the spreadsheet is incorrect. We will need to specify these individually.